ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.ts

```diff
@@ -177,6 +195,6 @@ export class ActivityRedesignedComponent extends ActivityComponent implements On
     }
 
     public isAgree(): boolean {
-        return this.model.activityCode === 'CONSENT' && this.agreeConsent && !this.model.readonly;
+        return this.model.activityCode === 'CONSENT' && this.agreeConsent;
```

Comment:
Not sure if we should hard-code an activity code into SDK. If we want to provide special treatment in SDK for consent activities (which this looks like) perhaps we we need to make these configurable? As we saw in Osteo, we can have more than one consent in a study.
Further: are we sure that this behavior is desirable or applicable for consent only? Wonder if it this is applicable for any multipart activity that has an agreement question.

Reply:
I don't like this approach too, but I thought we've already discussed that https://github.com/broadinstitute/ddp-angular/pull/122

Reply:
I think @yufengwng was suggesting we check the formType attribute. https://pepper-dev.datadonationplatform.org/docs#operation/getActivity CONSENT is one of three possible values and would deal with my hard-coding concern.

Reply:
Oh, right, my bad.

Reply:
I am confused about the `agreeConsent` @Input param. I can't figure out what is the meaning of it. Are you sure we need that? If you look at the list of input params for  the activity component, that param stands out as looking very different than the others. Appears to be only set to true via configuration for testboston.

Reply:
We need `agreeConsent` **@input** param to show agree buttons only for testboston study.
Otherwise, we will show agree button in other studies too.